omod/src/main/webapp/portlets/patientPrograms.jsp

```diff
@@ -468,7 +464,7 @@
 												<c:set var="stateStart" value="" />
 												<c:set var="retired" value="" />
 												<c:forEach var="state" items="${program.states}">
-													<c:if test="${!state.voided && state.state.programWorkflow.programWorkflowId == workflow.programWorkflowId && state.active}">
```

Comment:
Did you intentionally remove `state.active`?

Reply:
Yes. 
Users should be able to see states  even if they are closed/have an end date.